Sources/MFBActionBinding.h

```diff
@@ -0,0 +1,72 @@
+//
+//  MFBActionBinding.h
+//  Pods
+//
+//  Created by Nickolay Tarbayev on 28.07.2017.
+//
+//
+
+#import <Foundation/Foundation.h>
+
+#import "NSObject+MFBBindingAssertion.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface MFBActionBinding : NSObject
```

Comment:
I wonder if we could abuse UIControl in order to be able just drug "Value Changed" event to an IBAction.